core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1151,6 +1139,68 @@
 
   public static final String USE_KETTLE_DEFAULT = "false";
 
+  /**
+   * number of page per blocklet column
+   */
+  public static final String NUMBER_OF_PAGE_IN_BLOCKLET_COLUMN =
```

Comment:
all these newly added config is for V3 reader/writer, I think it is better to separte them into a separate Constrant class